group/iab/IAB_Nomcom.md

```diff
@@ -54,17 +54,50 @@ So that the Nomcom is not later surprised, it is important to point them to the
-- The chair of the IAB can choose when to call the question with the following conditions:
-    - The chair must not call the question any earlier than 1 week after the IAB is informed of the nominated candidate.
-    - The chair must not call the question prior to a teleconference which has this action as an agenda item.
-    - The chair must call the question by whatever deadline generally imposed - usually within 2 weeks after the IAB is advised of the nominated candidate.
-- A vote may be held by voice vote during a teleconference, by email ballot to the IAB members' mail list, or by secret ballot to the Executive Director and Chair as agreed to by consensus. If there is no consensus, vote will be by secret ballot to the Executive Director and Chair.
-- In the event that a sitting IAB member was nominated to replace the casual vacancy in the IESG, the IETF Executive Director would be informed of the casual vacancy only upon confirmation of the initial nomination, and the Nominations Committee would then need to undertake a subsequent nominations process for the IAB casual vacancy.
+
+The following process will be used by the IAB for confirmation of nominated
+candidates for vacant IESG positions:
+
+- The IAB will undertake its best endeavour to make a confirmation decision
+  according to this process within the stipulated time period allowed for
+  confirmation, and not allow the decision process to lapse. If there are
+  extenuating circumstances that do not permit the IAB to make a decision
+  within the necessary timeframe, then these circumstances shall be addressed
+  as appropriate.
+
+- There shall be a positive vote for confirmation. The confirmation is
+  considered an IAB decision and follows the process documented in
+  [IAB_Decisions](IAB_Decisions.md). If that process does not lead to a
+  positive vote, the candidate is rejected.
+
+- Usually the confirmation of the entire slate is brought to an IAB vote as a
+  single decision. However, if the IAB fails to reach consensus on the entire
+  slate, the IAB can split the vote into multiple votes, each for a portion of
+  the slate, so that some of the candidates can be confirmed.
+
+- The IETF chair is disqualified from voting or participating in the discussion
+  on the IESG slate or IESG candidates. The IAB may ask the IETF chair
+  questions about the needs of the IESG prior to the IAB being advised of a
+  nominated candidate.
+
+- If any voting IAB member is a candidate for an IESG position, they must
+  recuse themselves and are disqualified from voting or participating in the
+  discussion for that specific confirmaiton. The vote is split to allow these
+  recused members to participate in the vote to confirm other IESG positions.
+
+- In the event that a sitting IAB member was nominated to replace the casual
+  vacancy in the IESG, the IETF Executive Director would be informed of the
```

Comment:
I see that this is unchanged from before, but it is unclear why the IETF executive director needs to be informed?